samples/XAMLCatalog/XAMLCatalog/UIKitControls/UILabelViewController.h

```diff
@@ -17,5 +17,30 @@
 
 #import "MenuTableViewController.h"
 
-@interface UILabelViewController : MenuTableViewController
+@interface UILabelViewController : UIViewController <UITextFieldDelegate>
```

Comment:
>UIViewController [](start = 35, length = 16)

Using a generic viewcontroller vs a tableviewcontroller means as means we're not converging on the same pattern: adding a new control is not as simple as [addObject]. You also need to add a label explaining each control programmatically. It does save some space and workaround a tableview issue but it means our view controllers will look inconsistent.